sympy/series/tests/test_sequences.py

```diff
@@ -78,10 +78,10 @@ def test_SeqFormula():
     seq = SeqFormula(x*(y**2 + z), (z, 1, 100))
     assert seq.expand() == SeqFormula(x*y**2 + x*z, (z, 1, 100))
     seq = SeqFormula(sin(x*(y**2 + z)),(z, 1, 100))
-    assert seq.expand() == SeqFormula(sin(x*y**2)*cos(x*z) + sin(x*z)*cos(x*y**2), (z, 1, 100))
```

Comment:
I think it makes sense to include the default behavior (without additional arguments) in the test as well (but good that you added the trig=True case).

Same thing for power_exp below.

Reply:
I think the default behavior is when `trig=False`

Reply:
I should probably get rid of that then

Reply:
No, please keep both. There is a purpose to test both the behaviour of `trig=False` and that `trig=False` indeed is the default behaviour.

Reply:
Done